test/report.test.ts

```diff
@@ -27,6 +28,125 @@ describe("report", () => {
+                        },
+                    },
+                    {
+                        absoluteDateFilter: {
+                            dataSet: {
+                                uri: "/gdc/md/i6k6sk4sznefv1kf0f2ls7jf8tm5ida6/obj/330",
+                            },
+                            from: "2011-01-01",
+                            to: "2011-12-31",
+                        },
+                    },
+                    {
+                        relativeDateFilter: {
+                            to: 0,
+                            from: -3,
+                            granularity: "GDC.time.quarter",
+                            dataSet: {
+                                uri: "/gdc/md/myproject/obj/921",
+                            },
+                        },
+                    },
+                ];
+
+                const expectedShowFilters: ExecuteAFM.CompatibilityFilter[] = [
+                    {
+                        positiveAttributeFilter: {
+                            displayForm: {
+                                uri: "bar",
+                            },
+                            in: {
+                                uris: ["/gdc/md/bar1", "/gdc/md/bar2"],
+                            },
+                        },
+                    },
+                    {
+                        negativeAttributeFilter: {
+                            displayForm: {
+                                identifier: "foo",
+                            },
+                            notIn: {
+                                values: ["foo1", "foo2"],
+                            },
+                        },
+                    },
+                    {
+                        absoluteDateFilter: {
+                            dataSet: {
+                                uri: "/gdc/md/i6k6sk4sznefv1kf0f2ls7jf8tm5ida6/obj/330",
+                            },
+                            from: "2011-01-01",
+                            to: "2011-12-31",
+                        },
+                    },
+                    {
+                        relativeDateFilter: {
+                            dataSet: {
+                                uri: "/gdc/md/myproject/obj/921",
+                            },
+                            from: -3,
+                            granularity: "GDC.time.quarter",
+                            to: 0,
+                        },
+                    },
+                ];
+
+                const exportConfig: IExportConfig = {
+                    title: "title",
+                    format: "xlsx",
+                    mergeHeaders: false,
+                    showFilters,
+                };
+
+                return mockedReportModule()
+                    .exportResult(projectId, executionResult, exportConfig, { pollStep: 1 })
+                    .then(() => {
+                        const [, settings] = fetchMock.lastCall(
```

Comment:
I'm not sure this style is invalid `const [, settings]`, but please consider this `const [_, settings]`

Reply:
Hi @naibinh, Validator shows this error:

```
$ tsc --noEmit && yarn tslint && yarn prettier-check
test/report.test.ts:133:32 - error TS6133: '_' is declared but its value is never read.
133                         const [_, settings] = fetchMock.lastCall(
Found 1 error.
```